locales/en.js

```diff
@@ -0,0 +1,5 @@
+export default {
+    HomeHeaderRight: {
```

Comment:
Oh, I see, Lets change from

```json
{
    HomeHeaderRight: {
        title:  'Задачи'
    }
}
```
to

```json
{
    HomeHeaderRight: {
        IssuesTitle:  'Задачи'
    }
}
```

because it's title of Issues button